sk0-back/src/users/controllers/users.controller.ts

```diff
@@ -112,14 +117,20 @@ export class UsersController {
     description: "The found pathed by id",
     type: UpdateUserDto
   })
+  @UseGuards(AuthGuard("jwt"))
   async updateUserById(
     @Param("id") id: string,
```

Comment:
Not sure if that is intentional or not, but I think we should restrict the parameter in the function signature to be number(-ish) only. Instead of string we might wanted to have a `number` type here. Not sure how the tests should be rearranged.

Reply:
Hmmm, it was like this when I got my hands on it, so I'm not sure what the thought process was behind it.